Mathlib/Topology/Homeomorph.lean

```diff
@@ -566,12 +566,21 @@ def homeomorphOfUnique [Unique α] [Unique β] : α ≃ₜ β :=
 
 end
 
+/-- `Equiv.piCongrLeft` as a homeomorphism. -/
```

Comment:
Can you recall in this docstring what `Equiv.piCongrLeft` is? In fact, it would be good if the docstring for this one and the two next ones were constructed similarly.